config/locales/be/organisations.yml

```diff
@@ -438,16 +438,16 @@ be:
     jobs_contracts: Праца і кантакты
     latest_from: Апошняе ад %{title}
     notices:
-      changed_name: "%{title} зараз завецца <a href='%{link_href}'>%{link_text}</a>"
-      devolved: "%{title} з'яўляецца органам <a href='%{link_href}'>%{link_text}</a>"
+      changed_name_html: "%{title} зараз завецца <a href='%{link_href}'>%{link_text}</a>"
```

Comment:
Like with the [previous PR](https://github.com/alphagov/government-frontend/pull/2305#discussion_r759984448), could you explain the benefit of converting these to `_html`?

Reply:
Good point - this commit was older than the one you originally commented on.